packages/editor/src/dataviews/store/private-actions.ts

```diff
@@ -164,15 +174,18 @@ export const registerPostTypeSchema =
 			postTypeConfig.supports?.thumbnail &&
 				currentTheme?.[ 'theme-supports' ]?.[ 'post-thumbnails' ] &&
 				featuredImageField,
-			titleField,
+			! isDeisgnPostType &&
+				postTypeConfig.supports?.title &&
+				( postType === 'page' ? pageTitleField : titleField ),
+			postType === TEMPLATE_POST_TYPE && templateTitleField,
 			postTypeConfig.supports?.author && authorField,
 			statusField,
 			dateField,
 			slugField,
 			postTypeConfig.supports?.[ 'page-attributes' ] && parentField,
 			postTypeConfig.supports?.comments && commentStatusField,
-			passwordField,
 			templateField,
+			! isDeisgnPostType && passwordField,
```

Comment:
Same here about "design post type". It looks like the only changes we need in this file are related to post & pages post types — the only ones that use `usePostFields`. Scoping the changes to that would be good. If we want to prepare it for future usage in patterns & templates that's also fine. I guess I don't understand why we are introducing the navigation post type here.

Reply:
That was for the future, as I initially started by thinking I could move all fields at once. I'll remove them to reduce the scope.